### PR TITLE
Added UG documentation for TabView and AvatarView features.

### PR DESCRIPTION
**Pull request by KishoreJey**
Prepared UG document for the following features : 

-Scroll button customization in Maui Tab view
-Indicator stroke thickness in Maui Tab view
-Font auto-scaling mode in Maui Avatar view

### CODE REVIEW

File: MAUI/Avatar-view/customization.md [hunk 1]
```diff
@@ -466,3 +466,30 @@ this.Content = mainGrid;
 
 ![SfAvatarView content padding](Customization_Images/ContentPadding_AvatarView.png)
 
+### Font auto scaling enabled
+
+The FontAutoScalingEnabled property is used to automatically scale the avatar view's font size based on the operating system's text size. The default value of the `FontAutoScalingEnabled` property is `false.`
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Avatar-view/customization.md:471]: The `FontAutoScalingEnabled` property is used to automatically scale the avatar view's font size based on the operating system's text size. The default value of the `FontAutoScalingEnabled` property is `false.`

File: MAUI/Avatar-view/customization.md [hunk 2]
```diff
@@ -466,3 +466,30 @@ this.Content = mainGrid;
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<Grid>
+   <sfavatar:SfAvatarView  FontAutoScalingEnabled = true>
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Avatar-view/customization.md:478]: Please check the code snippet

File: MAUI/Avatar-view/customization.md [hunk 3]
```diff
@@ -466,3 +466,30 @@ this.Content = mainGrid;
...
+    </sfavatar:SfAvatarView>
+</Grid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+Grid mainGrid = new Grid();
+SfAvatarView avatarview = new SfAvatarView();
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Avatar-view/customization.md:487]: please use the proper name, avatarView

File: MAUI/TabView/Selection-Indicator-Customization.md [hunk 4]
```diff
@@ -180,4 +180,21 @@ You can customize the corner radius of the selection indicator using the [Indica
 
 {% endtabs %} 
 
-![Selection Indicator Corner Radius.](images/IndicatorCornerRadius.png) 
\ No newline at end of file
+![Selection Indicator Corner Radius.](images/IndicatorCornerRadius.png) 
+
+## Indicator's Stroke Thickness
+You can customize the stroke thickness of the selection indicator using the [IndicatorStrokeThickness] property in the Tab View.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Selection-Indicator-Customization.md:186]: Highlight the sentence properly, `IndicatorStrokeThickness`

File: MAUI/TabView/Selection-Indicator-Customization.md [hunk 5]
```diff
@@ -180,4 +180,21 @@ You can customize the corner radius of the selection indicator using the [Indica
...
+
+{% tabs %}
+
+{% highlight xaml %}
+    <tabView:SfTabView IndicatorStrokeThickness ="7">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Selection-Indicator-Customization.md:191]: check the code properly, end tag is missing

File: MAUI/TabView/Swiping.md [hunk 6]
```diff
@@ -27,4 +27,40 @@ The `EnableSwiping` property of `SfTabView` allows users to switch between tab c
 
 ### Limitations
 
-* Interference Between Child Controls and TabView Swiping: When a child control within a TabView supports horizontal swiping or interaction (e.g., a horizontal ScrollView, a custom swipe-enabled control, or a carousel), it can interfere with the TabView's touch gesture. This may result in unintended behavior, such as the TabView swiping when the child control is meant to handle the gesture, or vice versa. The overlapping gestures can cause confusion and disrupt the expected user experience, leading to a less intuitive interface.
\ No newline at end of file
+* Interference Between Child Controls and TabView Swiping: When a child control within a TabView supports horizontal swiping or interaction (e.g., a horizontal ScrollView, a custom swipe-enabled control, or a carousel), it can interfere with the TabView's touch gesture. This may result in unintended behavior, such as the TabView swiping when the child control is meant to handle the gesture, or vice versa. The overlapping gestures can cause confusion and disrupt the expected user experience, leading to a less intuitive interface.
+
+### Scroll Button Background Color
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Swiping.md:32]: Change the title to Scroll button customization. Add a common method for both the scroll button foreground and background color.

File: MAUI/Avatar-view/customization.md (hunk 1 above)
Comment (KishoreJey) [reply MAUI/Avatar-view/customization.md]: I have highlighted the sentence properly.

File: MAUI/Avatar-view/customization.md (hunk 2 above)
Comment (KishoreJey) [reply MAUI/Avatar-view/customization.md]: I have checked and updated the code snippet.

File: MAUI/Avatar-view/customization.md (hunk 3 above)
Comment (KishoreJey) [reply MAUI/Avatar-view/customization.md]: I have updated the name properly.

File: MAUI/TabView/Selection-Indicator-Customization.md (hunk 4 above)
Comment (KishoreJey) [reply MAUI/TabView/Selection-Indicator-Customization.md]: I have highlighted the sentence properly.

File: MAUI/TabView/Selection-Indicator-Customization.md (hunk 5 above)
Comment (KishoreJey) [reply MAUI/TabView/Selection-Indicator-Customization.md:191]: I have updated the code using end tag properly.

File: MAUI/TabView/Swiping.md (hunk 6 above)
Comment (KishoreJey) [reply MAUI/TabView/Swiping.md]: I have changed the title to scroll button customization and add a common method both the scroll button foreground and background color.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**